autogen/main/README.md

```diff
@@ -211,9 +211,8 @@ The node_pools variable takes the following parameters:
 | key | The key required for the taint | | Required |
 | logging_variant | The type of logging agent that is deployed by default for newly created node pools in the cluster. Valid values include DEFAULT and MAX_THROUGHPUT.  | DEFAULT | Optional |
 | local_ssd_count | The amount of local SSD disks that will be attached to each cluster node and may be used as a `hostpath` volume or a `local` PersistentVolume.  | 0 | Optional |
-{% if beta_cluster %}
```

Comment:
Looks like `local_ssd_ephemeral_count` is still beta in the provider, so let's keep the `if beta_cluster / endif` around it.